### PR TITLE
Add Footprint plugin.

### PR DESCRIPTION
**Pull request by ADmad**
Btw we should remove "plugin" from all link titles. It's quite unnecessary imo.

### CODE REVIEW

**Comment by dereuromark** [comment]
The titles need this information as not all are plugins. Some might be other resources etc.

File: README.md [hunk 1]
```diff
@@ -181,10 +181,11 @@ Focus is on the specific task.
 - [Geo plugin](https://github.com/dereuromark/cakephp-geo) - Containing [Geocoder behavior](http://www.dereuromark.de/2012/06/12/geocoding-with-cakephp/) and [GoogleMapsV3 helper](http://www.dereuromark.de/2010/12/21/googlemapsv3-cakephp-helper/).
 - ~~[Geocoder plugin](https://github.com/martinbean/cakephp-geocoding-plugin) - A lightweight geocoding plugin.~~
 
-## Logging
-*Plugins for generating and working with log files.*
+## Auditing / Logging
+*Plugins for auditing and logging.*
```
Comment (dereuromark) [inline README.md]: Do the hash links generate correctly from the overview index?

**Comment by ADmad** [comment]
Majority of the description after each title state it's a plugin. Even in case where it's not mentioned users can easily figure it out once they visit the repo.

File: README.md (hunk 1 above)
Comment (ADmad) [inline README.md]: Fixed

**Comment by dereuromark** [comment]
Hehe, that is sure true.

**Comment by dereuromark** [comment]
It is up to each person to make a better naming:
e.g.:

    Allows passing currently logged in user to model layer.

**Comment by ADmad** [comment]
@dereuromark I agree, I just copied the repo title/description. Fix it @jadb :stuck_out_tongue:

**Comment by jadb** [comment]
@ADmad, lol - that's description used on repo (people might not know its a CakePHP plugin), on this list, they know better - so u fix it, haha!

**Comment by ADmad** [comment]
@jadb Slacker :P

**Pull request merged**